src/SavedSearch.php

```diff
@@ -268,6 +268,21 @@ public function canCreateItem(): bool
         return parent::canCreateItem();
     }
 
+    /**
+     * Check if user can view saved searches
+     * Allow access in helpdesk interface for private searches
+     */
+    public static function canView(): bool
+    {
+        // Allow access in helpdesk interface (private searches only)
+        if (Session::getCurrentInterface() === 'helpdesk') {
+            return true;
+        }
+
+        // Standard check for central interface
+        return Session::haveRight(static::$rightname, READ);
```

Comment:
A profile using the standard interface typically has more permissions than a profile using the simplified interface. Following this logic, if access is always granted for simplified profiles, shouldn’t it always be granted for standard profiles as well?